### PR TITLE
Add healthcheck endpoint

### PR DESCRIPTION
**Pull request by carlpett**
This PR adds the possibility of having a healthcheck endpoint with the new flag `-web.health-path`. If not configured, the feature is disabled.

We use Consul for healthchecking and would like to not point it at /metrics. That would both add extra unnecessary calls to varnishstat, and also the output is added to the healthcheck status, which gets a bit messy. 

As I note in a comment, the only thing this actually shows at the moment is that the http server works and processes requests. If the `todo` noted in the `up` metric is resolved, syncronizing the healthcheck with that would be great.

### CODE REVIEW

**Comment by jonnenauha** [comment]
Oops, I put my comment into the merge commit message... Well here is what I said:

> cool thanks :) do you need a binary release from me for this to be useful, or you can do that yourself?

File: main.go [hunk 1]
```diff
@@ -87,6 +89,9 @@ func init() {
 	if len(StartParams.Path) == 0 || StartParams.Path[0] != '/' {
 		logFatal("-path cannot be empty and must start with a slash '/', given %q", StartParams.Path)
 	}
+	if len(StartParams.HealthPath) != 0 && StartParams.HealthPath[0] != '/' {
```
Comment (jonnenauha) [inline main.go]: could maybe do a sanity check here on top `StartParams.Path == StartParams.HealthPath` error out

Comment (carlpett) [inline main.go]: Good point! I'll add that.

**Comment by carlpett** [comment]
Regarding binary release that's no problem, we build rpms from the source. A release tag would be nice, though, if possible (after I fix the sanity check you mentioned)

**Comment by jonnenauha** [comment]
I did a tag + binary release. Might as well with latest go 1.7.3 and your nice feature o/

https://github.com/jonnenauha/prometheus_varnish_exporter/releases/tag/1.1

**Pull request merged**